### PR TITLE
K8SPG-135 document imagePullPolicy option

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Review by spron-in — APPROVED** [review]

File: source/glossary.rst [hunk 1]
```diff
@@ -93,6 +93,22 @@ The Operator options glossary
 
       :ref:`pmm.image<pmm-image>`
 
+   imagePullPolicy
```
Comment (hors) [inline source/glossary.rst:96]: @spron-in  I have re-check and we do not support it for now at all. I am going to create task about it,

File: source/glossary.rst [hunk 2]
```diff
@@ -93,6 +93,22 @@ The Operator options glossary
...
+
+      :ref:`backup.imagePullPolicy<backup-imagepullpolicy>`
+
+      :ref:`pgBadger.imagePullPolicy<pgbadger-imagepullpolicy>`
+
+      :ref:`pgBouncer.imagePullPolicy<pgbouncer-imagepullpolicy>`
+
+      :ref:`pgPrimary.imagePullPolicy<pgprimary-imagepullpolicy>`
+
+      :ref:`pgReplicas.REPLICA_NAME.imagePullPolicy<pgreplicas-imagepullpolicy>`
```
Comment (hors) [inline source/glossary.rst:108]: We will not have separate option for `pgReplicas`. The `imagePullPolicy` from `pgPrimary` will be used for `pgReplicas`.

**Review by hors — CHANGES_REQUESTED** [review]

File: source/glossary.rst (hunk 2 above)
Comment (hors) [reply source/glossary.rst:108]: @fiowro it was fixed and now we can merge it. So, please fix conflicts.

**Review by spron-in — APPROVED** [review]

**Review by spron-in — COMMENTED** [review]
Change the wording to "This option is used to set the policy for ..."

**Review by hors — APPROVED** [review]

**Pull request merged**